docs/nlu/using-bert.rst

```diff
@@ -0,0 +1,185 @@
+    in the pipeline can typically compensate for the lack of fine-tuning. 
+
+Run the Pipelines
+-----------------
+
+You can run both configurations yourself.
+
+.. code-block:: yaml
+
+    mkdir gridresults
+    rasa test nlu --config configs/config-light.yml \
+                  --cross-validation --runs 1 --folds 2 \
+                  --out gridresults/config-light
+    rasa test nlu --config configs/config-heavy.yml \
+                  --cross-validation --runs 1 --folds 2 \
+                  --out gridresults/config-heavy
+
+Results
+-------
+
+When this runs you should see logs appear. We've picked a few
+of those lines to list them here. 
+
+.. code-block:: none
+
+    # output from the light model
+    2020-03-30 16:21:54 INFO     rasa.nlu.model  - Starting to train component DIETClassifier
+    Epochs: 100%|███████████████████████████████| 50/50 [04:30<00:00, ...]
+    2020-03-30 16:23:53 INFO     rasa.nlu.test  - Running model for predictions:
+    100%|███████████████████████████████████████| 2396/2396 [01:23<00:00, 28.65it/s]
+    ...
+    # output from the heavy model
+    2020-03-30 16:47:04 INFO     rasa.nlu.model  - Starting to train component DIETClassifier
+    Epochs: 100%|███████████████████████████████| 50/50 [04:33<00:00,  ...]
+    2020-03-30 16:49:52 INFO     rasa.nlu.test  - Running model for predictions:
+    100%|███████████████████████████████████████| 2396/2396 [07:20<00:00,  5.69it/s]
+
+.. note::
+
+    From the logs we can gather an important observation. 
+    The heavy model consisting of ``BERT`` is a fair bit slower, not in training, but at inference time
+    we see a ~6 fold increase. Depending on your use-case this is something to seriously consider.
+
+The results from these two runs can be found in the ``gridresults`` folder. 
+We've summarised the main results below.
+
+Intent Classification Results
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the scores for intent classification.
+
+========  =========== =========== ===========
+ Config    Precision   Recall      f1 score
+========  =========== =========== ===========
+Light       0.7824      0.7819      0.7795
+Heavy       0.7894      0.7880      0.7843
+========  =========== =========== ===========
+
+Entity Recognition Results
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the scores for entity recognition.
+
+========  =========== =========== ===========
+ Config    Precision   Recall      f1 score
+========  =========== =========== ===========
+Light       0.7818      0.7282      0.7448
+Heavy       0.8942      0.7642      0.8188
+========  =========== =========== ===========
+
+Observations 
+~~~~~~~~~~~~
+
+On all fronts we see that the heavy model with the ``BERT`` embeddings performs better.
+More specifically, the performance gain for intent classification is marginal but it deserves mentioning that the effect
+is more pronounced in the case of entity recognition.
```

Comment:
"it deserves mentioning that the effect is more pronounced" is not friendly English :) I'm sure there's a simpler way to say this.